src/test/java/gov/healthit/chpl/aqa/stepDefinitions/FuzzyMatchTextSteps.java

```diff
@@ -25,7 +25,7 @@ public FuzzyMatchTextSteps() {
 
     @Given("^I am on listing details page \"([^\"]*)\"$")
```

Comment:
I think this step is a duplicate of one of the ones in a different step file, but with a slightly different name. Shouldn't re-invent that effort